src/Jobs/UpdateAllProducts.php

```diff
@@ -35,9 +37,56 @@ public function get_name(): string {
 	 * @return array
 	 */
 	public function get_batch( int $batch_number ): array {
+		return $this->get_filtered_batch()->get();
```

Comment:
Forgot to pass the `$batch_number` here.
```suggestion
		return $this->get_filtered_batch( $batch_number )->get();
```

Reply:
Thanks for catching that. This function is no longer actually used, but we can't remove it because the interface requires it to be implemented.